### PR TITLE
FHIR R4 DOCUMENATION BINARY

### PR DESCRIPTION
**Pull request by preethakarnan**
Description
----
Initiating the code review for Soarian FHIR R4 Binary resource.
PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] 
Binary1
Binary2
Binary3
Binary4
Binary5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/foundation/other/binary.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<%= json(:SOARIAN_R4_BINARY_READ_BY_ID) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Errors
+
+• The common [errors] and [OperationOutcomes] may be returned.
+ 
+[errors]: ../../#client-errors
+[OperationOutcomes]: ../../#operation-outcomes
```
Comment (DeepakGupta93) [inline content/soarian/r4/foundation/other/binary.md:61]: Hi Preetha,

For OperationOutcome link, we have used "https://hl7.org/fhir/R4/operationoutcome.html" in other resources, could you please check on this?

File: content/soarian/r4/foundation/other/binary.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## Retrieve by ID
+
+List an individual Binary resource by its ID:
+
+    GET /Binary/:id
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true) %>
```
Comment (khansasultana) [inline content/soarian/r4/foundation/other/binary.md:33]: is system Token not supported?

File: content/soarian/r4/foundation/other/binary.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+_Implementation Note_
+
+The `Binary.read` scope and the `DocumentReference.read` scope is required.
+
+### Headers
+
+<%= headers fhir_json: true %>
```
Comment (khansasultana) [inline content/soarian/r4/foundation/other/binary.md:41]: should these lines be included ?

DocumentReference.content.attachment.contentType should be used to set the Accept header.

This resource will not accept the application/json mime type unless the underlying binary data is json.

File: content/soarian/r4/foundation/other/binary.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+The search results include the following fields if they contain values:
+
+* [Binary id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Content type](https://hl7.org/fhir/r4/binary-definitions.html#Binary.contentType){:target="_blank"}
+* [Content](https://hl7.org/fhir/r4/binary-definitions.html#Binary.data){:target="_blank"}
+
+## Retrieve by ID
+
+List an individual Binary resource by its ID:
```
[the remark this answers is not in the document]
Comment (HRNWriter) [reply content/soarian/r4/foundation/other/binary.md:26]: @khansasultana, this is inconsistent with the other files. We are not referring to the resource itself in this case (_id), so this should have the standard term: ID.

PRODOC-160857.

**Review by HRNWriter — COMMENTED** [review]
See my comments.

PRODOC-160857.

File: content/soarian/r4/foundation/other/binary.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_BINARY_READ_BY_ID) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Errors
+
+• The common [errors] and [OperationOutcomes] may be returned.
```
Comment (khansasultana) [inline content/soarian/r4/foundation/other/binary.md:58]: since there is only point no need of bullet i guess.

Comment (khansasultana) [inline content/soarian/r4/foundation/other/binary.md:58]: since there is only point no need of bullet i guess.

**Review by khansasultana — CHANGES_REQUESTED** [review]

File: content/soarian/r4/foundation/other/binary.md (hunk 5 above)
Comment (preethakarnan) [reply content/soarian/r4/foundation/other/binary.md:58]: We have used bullet point commonly in all the files. To make it consistent with other resources we are using it here as well.

**Review by NeethuKP — APPROVED** [review]

**Review by DeepakGupta93 — APPROVED** [review]

File: content/soarian/r4/foundation/other/binary.md (hunk 3 above)
Comment (preethakarnan) [reply content/soarian/r4/foundation/other/binary.md:41]: Yes the underlying binary data is in json.

**Review by khansasultana — APPROVED** [review]

**Comment by slvsridhar** [comment]
After Publish : 
image
image
image

**Pull request merged**